tools/perf/brave_page_sets/brave_system_health/system_health_stories.py

```diff
@@ -47,6 +47,29 @@ def GenerateStoryDescription(cls):
     return 'Load %s' % cls.URL
 
 
+class _BraveMultiTabLoadingStory(_BraveLoadingStory):
+
+  TABS_COUNT = 10
+
+  def RunPageInteractions(self, action_runner):
+
+    def load_and_wait(action_runner):
+      action_runner.tab.WaitForDocumentReadyStateToBeComplete()
+      self._DidLoadDocument(action_runner)
+      action_runner.Wait(5)
```

Comment:
The delay is too large, resulting in 10 min for 10 retires.
Let's 1 sec delay between the tabs. Plus 5 sec delay after before the final measurement.

Reply:
Done.